Formula/metabase.rb

```diff
@@ -21,6 +21,8 @@ class Metabase < Formula
   def install
     if build.head?
       Language::Node.setup_npm_environment
+      # help yarn find npm's node-gyp binary inside superenv
+      ENV.prepend_path "PATH", "#{Formula["node"].opt_libexec}/lib/node_modules/npm/bin/node-gyp-bin"
```

Comment:
Also setting the correct global installation prefix to `~/.yarnrc` would help homebrew users to finally be able to install stuff globally with yarn, but it won't help formulas using yarn as a build dependency, because brew manipulates $HOME during installation and yarn would look in the wrong place again.

yarn installing node-gyp inside a formula environment:
```
error An unexpected error occurred: "EPERM: operation not permitted, symlink '../../../../../../private/tmp/metabase-20170726-52754-149xyg7/.brew_home/.config/yarn/global/node_modules/mkdirp/bin/cmd.js' -> '/usr/local/Cellar/node/8.2.1/bin/mkdirp'".
```